src/index/Vocabulary.cpp

```diff
@@ -101,38 +101,6 @@ void Vocabulary<S, C>::createFromSet(
   LOG(INFO) << "Done creating vocabulary.\n";
 }
 
-// _____________________________________________________________________________
-// TODO<joka921> is this used? if no, throw out, if yes, transform to
-// compressedString as key type
-template <class S, class C>
-template <typename, typename>
-ad_utility::HashMap<string, Id> Vocabulary<S, C>::asMap() {
-  ad_utility::HashMap<string, Id> map;
-  for (size_t i = 0; i < _words.size(); ++i) {
-    map[_words[i]] = i;
-  }
-  return map;
-}
-
-/*
-// _____________________________________________________________________________
-template <class S, class C>
-template <typename, typename>
-void Vocabulary<S, C>::externalizeLiterals(const string& fileName) {
-  LOG(INFO) << "Externalizing literals..." << std::endl;
-  auto ext = std::lower_bound(_words.begin(), _words.end(),
-                              EXTERNALIZED_LITERALS_PREFIX, _caseComparator);
-  size_t nofInternal = ext - _words.begin();
-  vector<string> extVocab;
-  while (ext != _words.end()) {
-    extVocab.push_back((*ext++).substr(1));
-  }
-  _words.resize(nofInternal);
-  _externalLiterals.buildFromVector(extVocab, fileName);
-  LOG(INFO) << "Done externalizing literals." << std::endl;
-}
- */
-
```

Comment:
Thanks for this, 
but I am currently cleaning up the `Vocabulary` module myself including a complete refactor.
Could you please delete the changes in these files for this PR, then I get less "merge conflicts" because of unused code, that both of us wanted to delete.

Reply:
To be honest I don't think I've ever gotten a merge conflict from deleted code where 2 parties wanted to delete it, but I can add it back if you prefer :D

Reply:
You are right, just leave it as is